_pages/update-the-config-file/understanding-baseurl.md

```diff
@@ -21,6 +21,6 @@ production it's `https://pages.18f.gov/guides-template{{page.url}}`.
 
 ### Change the `baseurl:` when serving locally
 
-If you you do change the `baseurl:` property in the `_config.yml` file,
+If you do change the `baseurl:` property in the `_config.yml` file,
```

Comment:
sorry that was not a real comment. just playing around with PRs I find in the `ruby-pull-requests` channel